tcms/testplans/static/testplans/js/get.js

```diff
@@ -253,6 +253,55 @@ function toolbarEvents() {
 
         sortTestCases();
     });
+
+    $('.js-toolbar-priority').click(function(ev) {
+        let selectedCases = getSelectedTestCases();
+
+        if (!selectedCases.length) {
+            alert($('#test_plan_pk').data('trans-no-testcases-selected'));
+            return;
+        }
+
+        for (let i = 0; i < selectedCases.length; i++) {
```

Comment:
```javascript
for (let/const testCase of selectedCases)
```
?

Reply:
selectedCases is an array of test case ids

Reply:
then why not
```javascript
for (let testCaseId of selectedCases)
```

Reply:
I get used to it.All the "FE" projects I have been involved so far used "for (i;i < ...)" for iterating arrays. I have no strong feelings. For objects prefer "for ... in". I guess it will work both ways.